File: jsprobelms/day4task/next_prime.js
// next prime number

num=1324

next=0

while(next<1){
    fact=0
    num=num+1
    for(i=2;i<num;i++){
        if(num%i==0){
            fact+=1
            break;
        }
    }
    if(fact==0){
        console.log(num,"is next prinme");
        next++
        
    }
}

File: pythonproblems/day1/nextprime.py
# next prime
num=int(input("enter a number:"))
next=0

while next<1:
    num=num+1
    fact=0

    for i in range(2, num):
        if num%i==0:
            fact+=1
            break
    if fact==0:
        print(num,"is next prime")
        next+=1